src/DetailsView/components/command-bar-buttons-menu.tsx

```diff
@@ -14,7 +12,6 @@ export type CommandBarButtonsMenuProps = {
     renderLoadAssessmentButton?: () => JSX.Element | null;
     getStartOverMenuItem: () => StartOverMenuItem;
     buttonRef: IRefObject<IButton>;
-    featureFlagStoreData?: FeatureFlagStoreData;
```

Comment:
@shanisebarona featureFlagStoreData was part of CommandBarProps before I removed it as part of this PR